transaction-status/src/parse_token/extension/confidential_mint_burn.rs

```diff
@@ -254,6 +254,25 @@ pub(in crate::parse_token) fn parse_confidential_mint_burn_instruction(
                 info: value,
             })
         }
+        ConfidentialMintBurnInstruction::ApplyPendingBurn => {
+            check_num_token_accounts(account_indexes, 1)?;
```

Comment:
`ApplyPendingBurn` expects the mint and the authority at the very least
```suggestion
            check_num_token_accounts(account_indexes, 2)?;
```